src/tools/pub/cmd/index.md

```diff
@@ -81,56 +85,28 @@ or any modification of the pubspec.
   Creates or updates the lock file, as needed.
 
 
-## Running and serving apps
-
-Pub supports development for running web apps and command-line apps.
-
-### Web apps
-
-If your web app's directory structure follows pub's [package layout
-conventions](/tools/pub/package-layout), you can use the pub development
-server ([`pub serve`](({{site.webdev}}/tools/pub/pub-serve))) to continuously
-build and serve the app's assets.
-This server can be accessed via the browser on localhost and
-allows you to view your web-based app.
+## Running command-line apps
 
-IDEs such as WebStorm also use `pub serve` to serve all of the
-Dart scripts and assets (HTML, images, CSS) to the browser.
-
-### Command-line apps
-
-There are two commands that you can use to run Dart scripts
-from the command line. Each command first runs any needed
-transformers before invoking the specified script.
+Two commands let you run Dart scripts from the command line:
 
 * The [`pub run`](/tools/pub/cmd/pub-run) command invokes a Dart script in your
   package, or in one of its dependencies.
 
 * The [`pub global`](/tools/pub/cmd/pub-global) command lets you work with
-  globally available packages that can be run when you are not currently inside
-  a package.
+  globally available packages.
 
-## Building and deploying apps and packages
+## Deploying packages and apps
```

Comment:
Hmm, do we want to have an aside here that redirects people to the new way of _building_ packages? "The pub tool has historically been used for building as well. That is no longer true. Learn here how this changed."

I'm sure people will be coming here trying to learn about building because there are people (and articles) that will claim that `pub` is Dart's build tool for some time still.